$OpenBSD: patch-luadoc_screen_lua,v 1.1 2013/11/05 07:33:33 dcoppa Exp $

commit 29ecc6095f6f566757d17538635eb2b7674ed96f
Author: Uli Schlachter <psychon@znc.in>
Date:   Thu Oct 31 10:31:54 2013 +0100

luadoc: Document screen outputs

This was introduced in 2009, commits 9393b2d1 and c08b9034.

Signed-off-by: Uli Schlachter <psychon@znc.in>

--- luadoc/screen.lua.orig	Sat Oct 12 18:48:49 2013
+++ luadoc/screen.lua	Tue Nov  5 08:23:23 2013
@@ -4,10 +4,14 @@
 module("screen")
 
 --- Screen is a table where indexes are screen number. You can use screen[1]
--- to get access to the first screen, etc. Each screen has a set of properties.
+-- to get access to the first screen, etc. Alternatively, if RANDR information
+-- is available, you can use output names for finding screen objects.
+-- Each screen has a set of properties.
 -- @field geometry The screen coordinates. Immutable.
 -- @field workarea The screen workarea.
 -- @field index The screen number.
+-- @field outputs If RANDR information is available, a list of outputs for this
+-- screen and their size in mm.
 -- @class table
 -- @name screen
 
